scripts/make_docs_locally.sh

```diff
@@ -5,10 +5,13 @@
 # python scripts/generate_dialect_comparison_docs.py
 
 UPDATE="TRUE"
-if [[ $UPDATE == "TRUE" ]]
-then
+if [[ $UPDATE == "TRUE" ]]; then
```

Comment:
Do we want to make this a flag that the user can set in the cmd?

Reply:
yea probably, I had thought about it but just got lazy